client/post-editor/editor-drawer/index.jsx

```diff
@@ -240,7 +242,23 @@ const EditorDrawer = React.createClass( {
 					isNew={ this.props.isNew }
 				/>
 			</AccordionSection>
-		)
+		);
+	},
+
+	renderSeo: function() {
+		if ( ! config.isEnabled( 'manage/advanced-seo' ) || ! this.props.site ) {
```

Comment:
Can we be more optimistic and allow the accordion to render even when site is not ready yet? For example, if I clear localStorage and load an editor page directly, this will not show the accordion heading at all until after sites have loaded.